libs/testserver/fake_workspace.go

```diff
@@ -0,0 +1,170 @@
+package testserver
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+// FakeWorkspace holds a state of a workspace for acceptance tests.
+type FakeWorkspace struct {
+	directories map[string]bool
+	files       map[string][]byte
+	// normally, ids are not sequential, but we make them sequential for deterministic diff
+	nextJobId int64
+	jobs      map[int64]jobs.Job
```

Comment:
Can we export the fields there? It's internal testing tool, not public SDK. It might be useful to print some of these during debugging.

Reply:
I think for debugging we can call ListJobs. I think it's important to have a clean interface because it will be tempting to rely on the internal state somewhere.